src/main/java/com/shop/model/ProductSpecification.java

```diff
@@ -18,13 +18,23 @@
     @ManyToOne(cascade = CascadeType.ALL)
     @JoinColumn(name = "specification_id")
     private Specification specification;
+    private int orderNumber;
```

Comment:
why do you introduce that field?

Reply:
to show specifications in a specific order